src/serverMain/kotlin/com/studo/campusqr/endpoints/ReportData.kt

```diff
@@ -95,4 +93,38 @@ suspend fun AuthenticatedApplicationCall.returnReportData() {
           impactedUsersEmailsCsvFileName = "${csvFilePrefix?.plus("-emails") ?: "emails"}.csv"
       )
   )
+}
+
+suspend fun AuthenticatedApplicationCall.listAllActiveCheckIns() {
+  if (!user.isModerator) {
+    respondForbidden()
+    return
+  }
+
+  val params = receiveJsonStringMap()
+  val emailAddress = params.getValue("emailAddress")
+
+  val checkIns = runOnDb {
+    val nextInfectionHours: Int = getConfig("nextInfectionHours")
```

Comment:
Please add a comment here and on the other place that we need to keep the check-out logic in sync when we plan to change it.